test/libsolidity/syntaxTests/parsing/conditional_with_assignment.sol

```diff
@@ -1,11 +1,10 @@
 contract A {
     function f() {
         uint y = 1;
-        uint x = 3 < 0 ? x = 3 : 6;
```

Comment:
Maybe there should be another test about this not working with ``x``?

Reply:
Actually, not sure why there was a ``y`` in the original test - maybe this was always meant to be ``y`` anyways...